src/uniprotkb/components/entry/__tests__/__snapshots__/EntryMain.spec.tsx.snap

```diff
@@ -7657,7 +7657,18 @@ exports[`Entry view should render for non-human entry 1`] = `
                         width="12.5"
                       />
                     </a>
-                    NZ_WKPS01000046.1
+                    <a
+                      class="external-link"
+                      href="https://www.ncbi.nlm.nih.gov/nuccore/%necleotideId"
```

Comment:
Looks like this pattern hasn't been replaced by a valid URL

Reply:
Good spot! Needed the database config file to be updated. Would be nice to devise a means of updating all mocks at once rather than copy and paste. Not sure how to do this given we also want TS applied to the mocks.

Reply:
Indeed, we don't want to rely on network everytime we do a test to pull the latest value.
Instead, eventually, we might want to have a script that we can call regularly (whenever backend tells us there was a change) that would generate those files that could be commited, like snapshots of the backend data, but used in our side a mocks for our own snapshots.